src/cmd/build.js

```diff
@@ -132,7 +174,10 @@ export async function defaultPackageCreator(
     filter: (...args) => fileFilter.wantFile(...args),
   });
 
-  let extensionName: string = manifestData.name;
+  let filenameTemplate = '{name}-{version}.zip';
```

Comment:
This ended up causing issues in the `defaultPackageCreator` test coverage.
Rather than adjust that test, I set `filename = DEFAULT_FILENAME_TEMPLATE` in `defaultPackageCreator` as well. Does that work?